### PR TITLE
feat: add autoplay banner

### LINKED ISSUE #42: Autoplay Banner

### PR DESCRIPTION
**Pull request by rizqitsani**
# Description & Technical Solution

- Add autoplay banner

# Checklist

- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] Already rebased against main branch.

# Screenshots

[Banner-Aether-Design-System.webm](https://user-images.githubusercontent.com/68275535/216077701-611aafae-aea8-43c5-ba1d-dfd0500a3acc.webm)

# Related issue

- Resolve #42

### CODE REVIEW

**Comment by theodorusclarence** [comment]
image

ini bisa bulet gitu ga yaa? biar bagus

**Comment by ppdbultimate** [comment]
#### 📝 Changed routes:


| [**/index.page**](https://design-system-moon-9nkwpzi3h-ppdbultimate.vercel.app/index.page) | [**/sandbox/banner.page**](https://design-system-moon-9nkwpzi3h-ppdbultimate.vercel.app/sandbox/banner.page) |
|:-:|:-:|
|<a href="https://design-system-moon-9nkwpzi3h-ppdbultimate.vercel.app/index.page">Screenshot of /index.page</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/gsqzVaSgl9oWr3FGJQD4RfO3sEInhif8dx3yyq_vswD7nTQSNG_1XssK5vbQiOW-zfyv6xMf3vCIr9ojcDuqJQ.png">(view full size)</a> |<a href="https://design-system-moon-9nkwpzi3h-ppdbultimate.vercel.app/sandbox/banner.page">Screenshot of /sandbox/banner.page</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/exqoL0Jo2nabGAihYc4InKHTzIFcGRyZ862OdhohWWfYlDcq7A8rbNWjEgImFQvYLA2wSWPLiTuYSPWkHhN5pA.png">(view full size)</a> |







Commit c35cad50dbba5a545dc5826d65e0664ef1b0b8d5 (https://design-system-moon-9nkwpzi3h-ppdbultimate.vercel.app).

**Review by theodorusclarence — APPROVED** [review]

File: src/constant/banner.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const BANNER_CONTENT = [
+  'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
```
Comment (winatungmiharja) [inline src/constant/banner.ts:2]: kedepannya konten banner mungkin ada yang bisa di-klik mas? seperti ada link/button gitu. Kalau iya mungkin extention filenya bisa diganti tsx mas

File: src/components/Banner.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+function PrevArrow({ onClick }: CustomArrowProps) {
+  return (
+    <IconButton
+      onClick={onClick}
+      icon={FiChevronLeft}
+      variant='ghost'
+      size='sm'
+      className='absolute top-1/2 -left-6 z-10 flex -translate-y-1/2 items-center rounded-full lg:-left-10'
+    />
```
Comment (winatungmiharja) [inline src/components/Banner.tsx:36]: kalau di mobile dia nanti collide kayak gini
image
yang `-left-6 lg:-left-10` bisa diganti `left-0 translate-x-full`

**Pull request merged**